cloudpickle/cloudpickle.py

```diff
@@ -833,6 +833,11 @@ def save_logger(self, obj):
 
     dispatch[logging.Logger] = save_logger
 
+    def save_root_logger(self, obj):
+        self.save_reduce(logging.getLogger, (), obj=obj)
+
+    dispatch[logging.RootLogger] = save_root_logger
```

Comment:
Since `logging.RootLogger` derives from `logging.Logger` I was wondering whether it would be worth to walk the class `__mro__` in order to find a custom reduce function. I went for the simpler option.